cypress/e2e/with_mock_data/systems.cy.ts

```diff
@@ -458,13 +458,13 @@ describe('Systems', () => {
       cy.findByRole('option', { name: 'Operational' }).click();
       cy.findByRole('button', { name: 'Save' }).click();
       cy.findByText(
-        'A System with the same name already exists within the same parent System. Please enter a different name.'
+        'A system with the same name already exists within the same parent System. Please enter a different name.'
       ).should('be.visible');
       cy.findByRole('button', { name: 'Save' }).should('be.disabled');
       cy.findByRole('button', { name: 'Cancel' }).click();
       cy.findByRole('button', { name: 'Add System' }).click();
       cy.findByText(
-        'A System with the same name already exists within the same parent System. Please enter a different name.'
+        'A system with the same name already exists within the same parent System. Please enter a different name.'
```

Comment:
```suggestion
        'A system with the same name already exists within the parent system. Please enter a different name.'
```
There are two instances of system and the same was removed.